### DISCUSSION TITLE
FATAL: password authentication failed for user "postgres"

### DISCUSSION THREAD

**Discussion by Dialecticus** [dbeaver/dbeaver#17200] [Q&A]
I don't know if this is a bug, or if I'm doing something wrong. I have PostgreSQL 11.5, and some databases in it. I used to be able to connect to them with DBeaver, but not anymore. I just updated to DBeaver 22.1.3 when I started getting the error in the title. This could be a coincidence though, because now I get this error even with older versions (22.0.5 for instance).

I can connect to the database with PSQL app. My password is accepted then. It just doesn't work with DBeaver anymore, for some reason.

Spec: Windows 10 64-bit, PostgreSQL 11.5, DBeaver any recent version, including the latest 22.1.3.

I can connect using PSQL console. My app can connect via ODBC. I can test ODBC connection for user 'postgres', and this also works. I can connect from DBeaver using ODBC. The only thing that doesn't work is native PostgreSQL connection from DBeaver. It used to work yesterday.

UPDATE: when I add my IP address to pg_hba.conf file, and then in DBeaver I connect to that IP address, instead of to localhost, it all works again. Makes no sense to me. I already have the default line in the .conf file, "host  all  all  127.0.0.1/32  md5", so I have no the feintest idea what's going on here.

**Comment by LonwoLonwo** [comment]
Hello @Dialecticus 
That's how PG driver works.
I am not sure that we can do something from DBeave's side.
I often saw advice like: "change localhost to 127.0.0.1 in the host field in the connection page". It sounds like magic, but usually, it works.
